requirements.txt

```diff
@@ -11,6 +11,7 @@ xhtml2pdf>=0.2.4
 sentry-sdk[flask]>=1.5.7,<2.7
 python3-saml>=1.15,<1.17
 google-auth>=2.6,<2.30
+nest-asyncio>=1.6.0
```

Comment:
Please don't use `nest-asyncio`, it is modifying the behaviour of the asyncio library. If something does not work with this library, we should address the reason and not patch asyncio to make it work.